llama_index/vector_stores/simple.py

```diff
@@ -147,6 +150,22 @@ def query(
                 similarity_top_k=query.similarity_top_k,
                 embedding_ids=node_ids,
             )
+        elif query.mode == MMR_MODE:
+            if "mmr_threshold" in kwargs:
```

Comment:
instead of this if/else, you can just do `mmr_threshold=kwargs.get("mmr_threshold", None)`

Reply:
Good point, will add that.